### ISSUE TITLE
[Bug]: MudExDialog and DefaultFocus

### ISSUE THREAD

**Issue by jddurand** [fgilde/MudBlazor.Extensions#109]
### Contact Details

jeandamiendurand@gmail.com

### What happened?

Default focus with MudExDialog does not work, while it is ok with MudDialog. C.f. discussion #108 .

Example code:

```
@page "/mudexdialanddefaultfocus"

@using MudBlazor.Extensions.Options

@rendermode @(new InteractiveServerRenderMode(prerender: false))

<MudDialogProvider></MudDialogProvider>

<MudExDialog Visible="true"
             DefaultFocus="DefaultFocus.None"
             OptionsEx="new DialogOptionsEx
             {
                 CloseButton=true,
                 BackdropClick=false,
                 Position = DialogPosition.Center
             }">
    <TitleContent>MudExDialog+MudTextField</TitleContent>
    <DialogContent>
        <MudTextField @bind-Value="TextValue">Text Field 1</MudTextField>
        <MudTextField @bind-Value="TextValue2" AutoFocus="true">Text Field 2 with AutoFocus</MudTextField>
    </DialogContent>
</MudExDialog>

@code
{
    public string TextValue { get; set; } = default!;
    public string TextValue2 { get; set; } = default!;
}
```
![image](https://github.com/user-attachments/assets/1acfd888-6ae4-48a5-a7df-0c7fea3e9d71)

### Expected Behavior

Default focus as per MudDialog

Example code:

```
@page "/mudexdialanddefaultfocus"

@rendermode @(new InteractiveServerRenderMode(prerender: false))

<MudDialogProvider></MudDialogProvider>

<MudDialog Visible="true"
           DefaultFocus="DefaultFocus.None"
           Options="new DialogOptions
           {
               CloseButton=true,
               BackdropClick=false,
               Position = DialogPosition.Center
           }">
    <TitleContent>MudDialog+MudTextField</TitleContent>
    <DialogContent>
        <MudTextField @bind-Value="TextValue">Text Field 1</MudTextField>
        <MudTextField @bind-Value="TextValue2" AutoFocus="true">Text Field 2 with AutoFocus</MudTextField>
    </DialogContent>
</MudDialog>

@code
{
    public string TextValue { get; set; } = default!;
    public string TextValue2 { get; set; } = default!;
}
```
Output:
![image](https://github.com/user-attachments/assets/9b0850a4-98d2-4cbd-8933-31f383f244b3)

### What application type are you referring to?

ServerRendered

### MudBlazor.Extension Version

2.0.7.1

### MudBlazor Version

7.15.0

### What .net Version are you using?

.Net8

### What browser are you using?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fgilde** [comment]
Thanks for reporting. I will have a look at it

**Comment by fgilde** [comment]
I fixed it. But version not released yet. However preview package with this fix is available [2.0.8-prev-2411231414-main](https://www.nuget.org/packages/MudBlazor.Extensions/2.0.8-prev-2411231414-main)

**Comment by jddurand** [comment]
Many thanks. Tested and sounds ok -;)

**Comment by fgilde** [comment]
your welcome. Version is released in [2.0.8](https://www.nuget.org/packages/MudBlazor.Extensions/2.0.8#versions-body-tab)
